GameOfThrones/Views/House/Index.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@model IEnumerable<GameOfThrones.Models.House>
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@{
+    ViewBag.Title = "Houses";
+}
+
+<h2>Houses</h2>
+<table class="table">
+    <tr>
+        <th>Name</th>
+        <th>Region</th>
+        <th>Words</th>
+        <th>Castle</th>
+        <th>Members</th>
+    </tr>
+    @foreach (var house in Model)
+    {
+        <tr>
+            <td>@Html.DisplayFor(field => house.Name)</td>
+            <td>@Html.DisplayFor(field => house.Region)</td>
+            <td>@Html.DisplayFor(field => house.Words)</td>
+            <td>@Html.DisplayFor(field => house.CastleId.Name)</td>
+            <td><input type="button" onclick="@("window.location.href='" + @Url.Action("Characters", "House") + "/" + house.HouseId + "'");" value="View members"></td>
+        </tr>
```

Comment:
Interesting more javascript/jquery method of submitting information.
Alternatively, a button bound to a `@Html.ActionLink` maybe an alternative potentially cleaner option here. E.g.
```
<button type="submit" value="chars" class="btn btn-default">@Html.ActionLink($"{house.Name} Characters","Characters","House",new { id = item.Id},null)
```